### PR TITLE
Rename annotations by users to comments in user interface

### LINKED ISSUE #1957: Change "annotations" in "comments"
The term "annotation" might seem OK in English, its counterpart "annotatie" in Dutch sounds very weird. Therefore I suggest to change the terminology in the UI into "comment/commentaar". Also results in more natural wording when talking about the annotations in documentation and other descriptive text.

### PR DESCRIPTION
**Pull request by chvp**
In dutch "Opmerking" was used. I chose to keep the annotation naming around for the collection of machine-generated annotations and comments, since this made the naming clearer (IMO).

Closes #1957.

### CODE REVIEW

File: config/locales/views/evaluations/nl.yml [hunk 1]
```diff
@@ -75,16 +75,16 @@ nl:
       feedback: "Feedback"
       grade: Score
       annotation_count:
-        zero: "Er werden geen annotaties toegevoegd aan je oplossing"
-        one: "Één annotatie"
-        other: "%{count} annotaties"
+        zero: "Er werdt geen opmerkingen toegevoegd aan je oplossing"
```
Comment (niknetniko) [inline config/locales/views/evaluations/nl.yml:78]:
```suggestion
        zero: "Er werden geen opmerkingen toegevoegd aan je oplossing"
```

**Review by niknetniko — APPROVED** [review]

File: config/locales/js/nl.yml [hunk 2]
```diff
@@ -109,16 +109,16 @@ nl:
       copy-to-clipboard: Kopieer code naar klembord
     user_annotation:
       meta: "%{user} · %{time}"
-      send: 'Annoteren'
+      send: 'Plaatsen'
```
Comment (bmesuere) [inline config/locales/js/nl.yml:112]: maybe this?
```suggestion
      send: 'Opmerking toevoegen'
```

File: config/locales/views/evaluations/nl.yml [hunk 3]
```diff
@@ -75,16 +75,16 @@ nl:
...
+        other: "%{count} opmerkingen"
       no_submission: "Je had geen tijdige oplossing voor deze oefening"
       invisible_total: "De totaalscore werd niet vrijgegeven"
       no_grading: "Deze oefening staat niet op punten"
     members_table:
       remove_user: Student verwijderen uit evaluatie
-      remove_user_consequences: "Als deze student al feedback gekregen had, dan zal deze verwijderd worden. Ben je zeker dat je deze student wil verwijderen?"
-      remove_users_consequences: "Als deze studenten al feedback gekregen had, dan zal deze verwijderd worden. Ben je zeker dat je deze studenten wil verwijderen?"
+      remove_user_consequences: "Als deze student al opmerkingen gekregen had, dan zullen deze verwijderd worden. Ben je zeker dat je deze student wil verwijderen?"
+      remove_users_consequences: "Als deze studenten al opmerkingen gekregen hadden, dan zullen deze verwijderd worden. Ben je zeker dat je deze studenten wil verwijderen?"
```
Comment (bmesuere) [inline config/locales/views/evaluations/nl.yml:87]: I think feedback is better here?

**Review by bmesuere — APPROVED** [review]
Maybe also get feedback from @pdawyndt here?

File: config/locales/js/nl.yml (hunk 2 above)
Comment (chvp) [reply config/locales/js/nl.yml:112]: This gets a bit long for a primary button, IMO. Maybe just "Toevoegen"?

File: config/locales/views/evaluations/nl.yml (hunk 3 above)
Comment (pdawyndt) [reply config/locales/views/evaluations/nl.yml:87]: Would also use feedback.

File: config/locales/js/nl.yml (hunk 2 above)
Comment (pdawyndt) [reply config/locales/js/nl.yml:112]: Google Doc also has "Reactie toevoegen" as tooltip.

Comment (chvp) [reply config/locales/js/nl.yml:112]: This is not a tooltip though, it's used here (the "Annoteren" button): 
screenshot_2021-08-03-174209

Comment (chvp) [reply config/locales/js/nl.yml]: I changed it to "Toevoegen" for now, re-requesting review to make sure this is OK.

**Review by bmesuere — APPROVED** [review]

**Pull request merged**